apps/cowswap-frontend/src/common/updaters/FeesUpdater.ts

```diff
@@ -61,27 +61,60 @@ function quoteUsingSameParameters(currentParams: FeeQuoteParams, quoteInfo: Quot
   } = currentParams
   const { amount, buyToken, sellToken, kind, userAddress, receiver, appData } = quoteInfo
   const hasSameReceiver = currentReceiver && receiver ? currentReceiver === receiver : true
+  const hasSameAppData = compareAppDataWithoutQuoteData(appData, currentAppData)
```

Comment:
Needed to drop the quote from appData otherwise it enters into an infinite loop.